src/Illuminate/Collections/Collection.php

```diff
@@ -1050,6 +1052,31 @@ public function splitIn($numberOfGroups)
         return $this->chunk(ceil($this->count() / $numberOfGroups));
     }
 
+    /**
+     * Get the first item in the collection, but only if exactly
+     * item exists. Otherwise, throw an exception.
+     *
+     * @param  callable|null  $callback
+     * @return mixed
+     *
+     * @throws ItemNotFoundException
+     * @throws MultipleItemsFoundException
```

Comment:
We always use fully qualified class names in DocBlocks.

Reply:
Ahh okay, sorry I wasn't aware of that. I'll get them changed to be fully qualified